tool/planet/etcd.go

```diff
@@ -773,14 +800,12 @@ loop:
 		case <-ticker.C:
 		}
 
-		procs, err := ps.Processes()
+		pid, err := systemctlGetPID(ctx, "etcd")
```

Comment:
If the goal is to be able to distinguish etcd running in planet vs etcd running inside a container, wouldn't checking `ppid == 1` to determine whether the process is the planet's etcd be more reliable? And simpler for that matter (just a slight change of existing condition).

My main concern with the implemented approach is that if we rely on some behavior based on source code findings ("from source code it looks like...", "from manual testing..." in the comments above), then this solution may be fragile and potentially break b/w systemd versions (esp. considering it's systemd...)